src/lib/components/ui/ArticleList.svelte

```diff
@@ -40,6 +40,20 @@
 		visibleArticles += ARTICLES_PER_PAGE;
 	}
 
+	function validateColor(color: string): string {
+		const colorRegex =
+			/^(#([0-9A-Fa-f]{3}|[0-9A-Fa-f]{6})|rgb[a]?\([\d\s%,.]+\)|hsl[a]?\([\d\s%,.]+\)|var\(--[\w-]+\)|transparent|inherit|[a-z]+)$/i;
+		return colorRegex.test(color) ? color : 'inherit';
+	}
```

Comment:
You don't need to validate the color here again, you're already doing that in the `SponsorFieldsSchema`